frontend/resourceadm/pages/ResourceDashboardPage/ResourceDashboardPage.tsx

```diff
@@ -76,8 +64,8 @@ export const ResourceDashboardPage = (): React.ReactNode => {
               {`Alle ressurser (${resourceListData?.length ?? 0})`}
             </Heading>
           </div>
-          <ResourceTable list={filteredTableData(resourceListData ?? [])} />
-          {filteredTableData(resourceListData ?? []).length === 0 && (
+          <ResourceTable list={filterTableData(searchValue, resourceListData ?? [])} />
+          {filterTableData(searchValue, resourceListData ?? []).length === 0 && (
```

Comment:
we can simplify this code by assigning the filter value to a variable. :)
```suggestion
          <ResourceTable list={filteredResourceList)} />
          {filteredResourceList.length === 0 && (
```